proposals/0002-gep.md

```diff
@@ -0,0 +1,51 @@
+---
+MEP: 0002
+Title: Unifed API
+Discussion: Link
+Implementation: Link
+---
+
+# TITLE: Unified API
+
+## Abstract
+
+This GEP discusses the unified API that handles the interaction with LLM providers.
+
+## Motivation
+
+[TBU, what problem we are trying to solve, focus on users, use cases, and all important context]
+
+The Unified API holds all the logic needed to create a request to a provider. This solution allows the clients to be easily
+envoked by a provider pool.
+
+
+### Requirements
+
+
+R1: Should be a Hertz Client
```

Comment:
You probably mean Hertz API endpoint, right?

Reply:
Yes

Reply:
This is mostly for me haha. I'll use hertz to handle requests in each provider.